### DISCUSSION TITLE
OpenProject and Dates not importing

### DISCUSSION THREAD

**Discussion by sgillf** [super-productivity/super-productivity#3775] [General]
Hi All, I've got my integration with openproject working well, with the exception of start and end dates. 

The "type", "Status", "Assignee" and durations are all working, but no matter how I play around with the start and end dates of the workpackages in openproject, I can't get it to populate in super-productivity.

Am I doing something wrong? Anybody else have this issue?

Having the dates be pulled in automatically would be an huge plus!

Thanks!

![image](https://github.com/user-attachments/assets/9063b1bb-c275-4579-bd9e-ca47a4709a10)

**Comment by johannesjo** [comment]
Hi there! That is a thing that is not integrated. PRs are welcome!

**Comment by sgillf** [comment]
Thanks! I'll give it a try!
